spicy/runtime/src/driver.cc

```diff
@@ -224,6 +227,15 @@ driver::ParsingState::State driver::ParsingState::_process(size_t size, const ch
         return Done;
     }
 
+#ifdef DEBUG
```

Comment:
Let's either use a proper way to figure out whether a debug logger is enabled, or execute this code unconditionally (the logging would still figure out whether debug logging is enabled).

Reply:
yeah, the latter is fine